zulipterminal/ui_tools/buttons.py

```diff
@@ -199,7 +200,8 @@ def __init__(self, user: Dict[str, Any], controller: Any,
         self.recipients = frozenset({self.user_id, view.model.user_id})
 
         super().__init__(controller,
-                         caption=user['full_name'],
+                         caption="(You) " + user['full_name'] if \
+                                  is_current_user else user['full_name'],
```

Comment:
This would be clearer with a separate local variable before the `super()` call; generally if a one-line conditional wraps like this, it's *probably* clearer defined elsewhere, particularly if variables are repeated.